tool/elk/images/install-elk.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -ex
+
+wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+sudo apt-get install apt-transport-https
+echo "deb https://artifacts.elastic.co/packages/7.x/apt stable main" | sudo tee /etc/apt/sources.list.d/elastic-7.x.list
+sudo apt-get update && sudo apt-get install elasticsearch kibana
+
+sudo systemctl daemon-reload
+
+es_config=$(mktemp)
+cat > $es_config << EOF
+node.name: logging.vaticle.com
+path.data: /var/lib/elasticsearch
+path.logs: /var/log/elasticsearch
+network.host: 0.0.0.0
+discovery.type: single-node
+http.port: 2053
+xpack.security.enabled: true
+xpack.security.http.ssl.enabled: true
+xpack.security.http.ssl.key: /etc/elasticsearch/key.pem
```

Comment:
Key & certificate **must** be in `/etc/elasticsearch`; this is mandated by policy of the JVM that runs ElasticSearch and throws `AccessDeniedException` if they are placed differently